concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md

```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
 
 ### Manual Configuration
 
-After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration.
+After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below, you will find instructions on how to set up a license key depending on whether you connect sources via `<script>` or use modules and bundlers.
 
-Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-license.ts`. Paste the license key you copied from the Download Manager:
+---
+##### jQuery
 
-    <!-- tab: devextreme-license.ts -->export const licenseKey = 'DEVELOPER_LICENSE_KEY’;
+#### Scripts
```

Comment:
```suggestion
#### NonModular
```

I suggest using 'Modular' and 'Non Modular' terms

In the end, modules are also bundled and added to a page with `<script>` tag.